database/src/main/java/com/firebase/ui/database/Timestamp.java

```diff
@@ -0,0 +1,25 @@
+package com.firebase.ui.database;
+
+import com.google.firebase.database.Exclude;
+import com.google.firebase.database.PropertyName;
+import com.google.firebase.database.ServerValue;
+
+import java.util.Map;
+
+public class Timestamp {
+    private long mTimestamp;
+
+    @PropertyName("timestamp")
+    private Map<String, String> getServerValue() {
+        return ServerValue.TIMESTAMP;
+    }
+
+    @Exclude
+    public long getTimestamp() {
+        return mTimestamp;
+    }
```

Comment:
Should I check to see if `mTimestamp` is 0 (null) here? If so, I would return the system time. Thoughts?